repondeur/tests/integration/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from selenium import webdriver
+from selenium.common.exceptions import WebDriverException
+from webtest.http import StopableWSGIServer
+
+
+@pytest.fixture#(scope="session")
```

Comment:
There is definitely something wrong here for now, either because the browser is caching some pages (fragments?) or because the `DBSession` is not flushed prior the browser request is issued. Especially visible with `test_visionneuse_detail` serie. To be continued.